build/images/Dockerfile.build.ubi

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.15 as antrea-build
+
+WORKDIR /antrea
+
+COPY go.mod /antrea/go.mod
+
+RUN go mod download
+
+COPY . /antrea
+
+RUN make antrea-agent antrea-controller antrea-cni antctl-ubuntu
```

Comment:
Would you rename antctl-ubuntu to antctl-linux?

Reply:
Done. Rename `antctl-ubuntu` target to `antctl-linux`